addons/painkillers/ACE_Medical_Actions.hpp

```diff
@@ -0,0 +1,20 @@
+class ACE_Medical_Actions {
+    class Basic {
+        class Morphine;
+        class PainKillers: Morphine {
```

Comment:
Can we prefix this? Would be safer if/when ACE3 adds it so we can just swap items we use. Also `Painkillers`.